server/app/views/TranslationUtils.java

```diff
@@ -0,0 +1,67 @@
+package views;
+
+import com.google.auto.value.AutoValue;
+import com.google.common.base.Splitter;
+import java.util.List;
+import java.util.UUID;
+import play.i18n.Messages;
+import services.MessageKey;
+
+/** Utility class for managing translated content. */
+public final class TranslationUtils {
+  /**
+   * Helper to split a translated string with a single argument into the components preceeding the
+   * argument and the components after the argument.
+   *
+   * <p>This makes it possible to visually style subcomponents of the translated message separately
+   * when different languages may put the argument in a different position within the translated
```

Comment:
This won't work for RtL languages I think, though I don't think we currently need to or do it otherwise.  Can you make an issue and a todo so it's easier to find when we need to support one?

Reply:
I thought I replied, but don't see it on this thread..

What was the outcome? 

My concern was that parts 1 and 2 could get flipped in a RtL language such that we'd highlight the opposite part.

Adding a bug is fine.

Reply:
I still think my June 16th comment is relevant for this question. Regarding the specific scenario of part 1 and part 2 getting flipped in a RtL language, I don't think that would be an issue since we only have two components:
  1) The "static" text
  2) The dynamic value

We would apply visual styling to one of these but not the other and the helper accounts for the dynamic value being in a different location (e.g. before instead of after) in an RtL language.

I'm not sure what to put in the bug since I don't actually think that this is introducing a new issue for languages of that sort.